CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Translations](#translations)
+
+## Feature Requests
+
+Feature requests should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Ensure
+the feature request was not already reported by searching the issue tracker
+and/or mailing list. Feature requests are filed under the `enhancement` label
+on the [issue tracker](https://github.com/newsboat/newsboat/labels/enhancement).
+
+
+## Reporting Bugs
+
+**Do not open a GitHub issue if the bug is a security vulnerability**. Send an
+email to security@newsboat.org. Please encrypt your emails to 
+[PGP key 4ED6CD61932B9EBE](https://newsboat.org/newsboat.pgp)
+
+Bug reports should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Again,
+please ensure the bug was not already reported.
+
+
+## Patches / Pull Requests
+
+Patches can be submitted by sending a 
```

Comment:
I think this paragraph should be the last in the section. Other paragraphs explain *what* to work on and *how* to "package" the work, and this one will explain the final point, i.e. how to submit the work.

Reply:
I think you misunderstood me here, @amritbrar, and moved the whole *section* to the end of the *document*. I think I'll just make a commit here myself, and we can revert it if there's disagreement over it.